controllers/sriovibnetwork_controller_test.go

```diff
@@ -190,3 +190,17 @@ var _ = Describe("SriovIBNetwork Controller", func() {
 		})
 	})
 })
+
+func generateExpectedIBNetConfig(cr *sriovnetworkv1.SriovIBNetwork) string {
```

Comment:
why was it moved from test/utils ?

@fedepaol  @SchSeba do you know if test/utils module being used in other project ?